codesign/test/file_codesign_visitor_test.dart

```diff
@@ -21,16 +21,17 @@ void main() {
   const String codesignTeamIDFilePath = '/tmp/teamID.txt';
   const String inputZipPath = '/tmp/input.zip';
   const String outputZipPath = '/tmp/output.zip';
-  final MemoryFileSystem fileSystem = MemoryFileSystem.test();
   final List<LogRecord> records = <LogRecord>[];
 
+  late MemoryFileSystem fileSystem;
   late FakeProcessManager processManager;
   late cs.FileCodesignVisitor codesignVisitor;
-
-  Directory rootDirectory = fileSystem.systemTempDirectory.createTempSync('conductor_codesign');
+  late Directory rootDirectory;
 
   group('test reading in passwords: ', () {
     setUp(() {
+      fileSystem = MemoryFileSystem.test();
```

Comment:
This doesn't block your PR, but in the future you can remove a lot of this duplicate work in a refactor by nesting your setUps.

It's common to have a group for the entire file, then groups for different parts of logic

Reply:
Thanks a lot for the suggestion! Refactored